src/bop_interfaces/listbox.rs

```diff
@@ -78,6 +78,10 @@ impl ListBox {
         pos + (self.page * self.screen.get_height() as usize)
     }
 
+    pub fn set_position(&mut self, pos: usize) {
```

Comment:
I didn't quite understand why this function method was needed. For API consistency? You can just ```listbox.position``` for getting or setting values

Reply:
Oh, I assumed that was a private property, since it's something managed internally by other methods

Reply:
Okay, i fix that